pom.xml

```diff
@@ -62,7 +62,8 @@
         <check.skip-dependency-versions>true</check.skip-dependency-versions>
         <check.spotbugs-exclude-filter-file>${main.basedir}/spotbugs-exclude.xml</check.spotbugs-exclude-filter-file>
         <killbill-api.version>0.54.0-e27330f-SNAPSHOT</killbill-api.version>
-        <killbill-client.version>1.3.0-fc92bc1-SNAPSHOT</killbill-client.version>
+        <killbill-client.version>1.3.0-81035b1-SNAPSHOT</killbill-client.version>
```

Comment:
I updated those but because I merged my PR, you should accept the one from the `work-for-release-0.23.x` - only keep the `killbill-plugin-api.version` below. You probably need to resync with  `work-for-release-0.23.x`

Reply:
Done